tasks/rbd_fsx.py

```diff
@@ -52,7 +52,18 @@ def _run_one_client(ctx, config, role):
     args.extend([
         'adjust-ulimits',
         'ceph-coverage',
-        '{tdir}/archive/coverage'.format(tdir=testdir),
+        '{tdir}/archive/coverage'.format(tdir=testdir)
+    ])
+
+    if config.get('valgrind'):
+        args = teuthology.get_valgrind_args(
+            testdir,
+            'fsx_{id}'.format(id=role),
+            args,
+            config.get('valgrind')
+        )
```

Comment:
jason points out that ceph task's valgrind_post() checks all valgrind logs regardless of ceph task configuration